src/drive/ducks/files/Toolbar.jsx

```diff
@@ -45,6 +46,22 @@ class Toolbar extends React.Component {
     const notRootfolder = displayedFolder && displayedFolder.id !== ROOT_DIR_ID
     return (
       <div className={styles['fil-toolbar-files']} role="toolbar">
+        <IntentButton
+          className={classNames(
+            styles['c-btn'],
+            styles['c-btn--regular'],
+            styles['u-hide--mob']
+          )}
+          intent={{
```

Comment:
I love the idea of a having a component for intent, however I think we should keep the signature of the method [[https://github.com/cozy/cozy-client-js/blob/master/docs/intents-api.md#intents|`cozy.client.intents.create`]]
Two reasons:
* It's clearer to follow the parameter
* The given intent here is not an existing intent from DB, it does not have and id for example. I think we should keep the concept of an intent resulting from action, doctype and data, and not mocked object with only a part of its attributes.
So I suggest to do:
```jsx
<IntentButton
  action='CREATE'
  doctype='io.cozy.accounts'
  data={{
    dataType: 'bill'
  }}
>
```
What do you think ?
What do @CPatchane and @y-lohse think ?

Reply:
👍 that makes sense

Reply:
Agree, it's more coherent too 👍

Reply:
:+1: